commerce-media-dl/build.gradle

```diff
@@ -10,4 +9,6 @@ dependencies {
 	compileOnly group: "org.osgi", name: "org.osgi.core", version: "5.0.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
 	compileOnly project(":apps:commerce:commerce-api")
+	compileOnly project(":apps:commerce:commerce-media-api")
+	compileOnly project(":apps:commerce:commerce-service")
```

Comment:
@alecsloan , As @ethanbustad says, we have to avoid this dependency as well.  commerce media will be a base module, so must not have dependencies with other commerce modules at all.